image_writer.py

```diff
@@ -0,0 +1,80 @@
+"""Takes in image text (top text and bottom text) and an image, and returns an image with text written on it in meme
+style.
+"""
+
+from PIL import Image, ImageDraw, ImageFont
+from random import randint
+
+
+def write_text_to_image(text_top, text_bottom, image_name):
+    """Main function. Writes inputted text to image in 'meme' style"""
+    image = _import_image(image_name)
+    # processed_image = _process_image
+    font, text_color, spacing = _determine_text_attributes()
+    draw = ImageDraw.Draw(image)
+    text_top_x, text_top_y, text_bottom_x, text_bottom_y = _determine_text_placement(draw, image, font, text_top,
+                                                                                     text_bottom)
+    _draw_image(draw, text_top, font, text_color, text_top_x, text_top_y, spacing)
+    _draw_image(draw, text_bottom, font, text_color, text_bottom_x, text_bottom_y, spacing)
+    _save_image(image, image_name)
```

Comment:
The main goal of this module is to draw text onto an image. I would not require that the image be saved to a filename as part of that main process, and instead just return the PIL `Image` since it affords a ton of flexibility. The whole project main might write it to the network or have configurable formats or do something else with it.